server/pulp/server/managers/status.py

```diff
@@ -55,6 +58,7 @@ def get_broker_conn_status():
         conn.connect()
         conn.release()
         return {'connected': True}
-    except:
+    except Exception, e:
+        _logger.exception(e)
```

Comment:
My understanding of `logger.exception` is that it includes exc_info in addition to your message, so I think this might be redundant. Something like `_logger.exception('Connection to broker failed!')` might be better.